### ISSUE TITLE
(scheduling_dpmsolver_multistep.py) function: _convert_to_karras () sigma_min and sigma_max seems mix up?

### ISSUE THREAD

**Issue by yangdeai** [huggingface/diffusers#5077]
Describe the bug

When I print the sigma_min  and sigma_max  in scheduling_dpmsolver_multistep.py's  _convert_to_karras () , I find that sigma_min  is larger  than sigma_max ? Is it right?

```
import numpy as np
import torch

#diffusers
def convert_to_karras(num_inference_steps=10) -> torch.FloatTensor:
    """Constructs the noise schedule of Karras et al. (2022)."""

    # sigma_min: float = in_sigmas[-1].item()  # big num
    # sigma_max: float = in_sigmas[0].item()   # small num
    sigma_min = 1e-2  
    sigma_max = 80

    rho = 7.0  # 7.0 is the value used in the paper
    ramp = np.linspace(0, 1, num_inference_steps)
    min_inv_rho = sigma_min ** (1 / rho)
    max_inv_rho = sigma_max ** (1 / rho)
    sigmas = (max_inv_rho + ramp * (min_inv_rho - max_inv_rho)) ** rho
    return sigmas

#webui
def get_sigmas_karras(n, sigma_min, sigma_max, rho=7.):
    """Constructs the noise schedule of Karras et al. (2022)."""
    ramp = torch.linspace(0, 1, n)
    min_inv_rho = sigma_min ** (1 / rho)
    max_inv_rho = sigma_max ** (1 / rho)
    sigmas = (max_inv_rho + ramp * (min_inv_rho - max_inv_rho)) ** rho
    return sigmas



if __name__ == '__main__':
    sigma_min = 1e-2
    sigma_max = 80
    n = 10

    df = convert_to_karras(num_inference_steps=n)
    print(df)
    wb = get_sigmas_karras(n=n, sigma_min=sigma_min, sigma_max=sigma_max)
    print(wb)

```

### Reproduction

_convert_to_karras () sigma_min and sigma_max seems mix up?

![1694944728554_0EF6CF7E-0434-4adf-BB4E-BF7372CBF8B7](https://github.com/huggingface/diffusers/assets/55951120/f65447f0-4719-4ab3-970e-a714dd3f879c)

### System Info

diffusers: 0.20.0

### Who can help?

@patrickvonplaten ,@sayakpaul,@williamberman

**Comment by patrickvonplaten** [comment]
@yiyixuxu can you check this?

**Comment by yiyixuxu** [comment]
@yangdeai 

Thanks for reporting this! will be fixed in https://github.com/huggingface/diffusers/pull/4986
(note that this does not affect the results so feel free to continue using this scheduler as it is)

**Comment by yiyixuxu** [comment]
fixed here https://github.com/huggingface/diffusers/pull/4986
